### PR TITLE
Make check-in files available during Smoke-Test

### PR DESCRIPTION
**Pull request by felixtheflex**
This also involves configuring ruby

### Context

Make check-in files available during Smoke-Test and create a suitable ruby environment

### Changes proposed in this pull request

Make check-in files available during Smoke-Test and create a suitable ruby environment

### CODE REVIEW

**Review by darokel — APPROVED** [review]

**Comment by darokel** [comment]
[Fix register smoke test](https://trello.com/c/20MjMRX8/865-fix-register-smoke-test)

File: .github/workflows/smoke-test.yml [hunk 1]
```diff
@@ -11,13 +11,25 @@ jobs:
...
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+
+      - name: install bundle
+        run: |
+          gem install bundler
```
Comment (darokel) [inline .github/workflows/smoke-test.yml:24]: so do we want to install rspec separately here or do a bundle with a different Gemfile?

**Comment by felixtheflex** [comment]
[Make checkin files available during smoke test](https://trello.com/c/BNSgFsKs/862-make-checkin-files-available-during-smoke-test)

File: .github/workflows/smoke-test.yml [hunk 2]
```diff
@@ -11,13 +11,31 @@ jobs:
     name: smoke-tests-${{ github.event.inputs.environment }}
     runs-on: ubuntu-latest
     steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby 2.7.2
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.2
```
Comment (gpeng) [inline .github/workflows/smoke-test.yml:20]: Would be good to get this from the .tool-versions or Gemfile. Let's think about that when we upgrade Ruby next though.

File: Gemfile [hunk 3]
```diff
@@ -85,7 +85,9 @@ group :development, :test do
   gem "launchy"
 
   # Testing framework
+  gem "rspec"
```
Comment (gpeng) [inline Gemfile:88]: Do we need this. `rspec-rails` includes it I think.

**Review by gpeng — APPROVED** [review]
Can we fixup the commits before merging? There's a merge commit in there. 👍

File: Gemfile (hunk 3 above)
Comment (darokel) [reply Gemfile:88]: This is what's causing the issues when you try to run the tests locally. It needs to be bundled as the rails one brings in some additional extras which require more dependencies to be available.

Comment (darokel) [reply Gemfile]: You're right, i was able to get it working using `rspec-rails`

**Pull request merged**